src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLinear.cs

```diff
@@ -2,8 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Float = System.Single;
+
```

Comment:
I think @Zruty0 meant remove it altogether, but that's all right, we can do a sweep for this later.